docs/sources/enterprise/access-control/fine-grained-access-control-references.md

```diff
@@ -11,28 +11,32 @@ The reference information that follows complements conceptual information about
+| `fixed:server:admin:read`             | `server.stats:read`                                                                                                                                                                                                                                                          | Read server stats                                                                                                                         |
+| `fixed:settings:admin:read`           | `settings:read`                                                                                                                                                                                                                                                              | Read settings                                                                                                                             |
+| `fixed:settings:admin:edit`           | All permissions from `fixed:settings:admin:read` and<br>`settings:write`                                                                                                                                                                                                     | Update settings                                                                                                                           |
+| `fixed:datasource:editor:read`        | `datasources:explore`                                                                                                                                                                                                                                                        | Allows to access the **Explore** tab                                                                                                      |
+| `fixed:datasources:admin`             | `datasources:read`<br>`datasources:create`<br>`datasources:write`<br>`datasources:delete`                                                                                                                                                                                    | Allows to create, read, update, delete data sources.                                                                                      |
+| `fixed:datasources:id:viewer`         | `datasources:id:read`                                                                                                                                                                                                                                                        | Allows to read data sources ID.                                                                                                           |
```

Comment:
Minor: I think `data source IDs` sounds slightly better?

Reply:
```suggestion
| `fixed:datasources:id:viewer`         | `datasources:id:read`                                                                                                                                                                                                                                                        | Allows to read data source IDs.                                                                                                           |
```

Reply:
Could it be `data sources' IDs`? Because each data source has 1 ID?

Reply:
```suggestion
| `fixed:datasources:id:viewer`         | `datasources:id:read`                                                                                                                                                                                                                                                        | Allows to read data source IDs.                                                                                                           |
```